server/ai_live_video.go

```diff
@@ -66,7 +66,11 @@ func startTricklePublish(ctx context.Context, url *url.URL, params aiRequestPara
 			cancel()
 			return
 		}
-		thisSeq, atMax := slowOrchChecker.BeginSegment()
+		thisSeq, atMax, sinceLastBeginSegment := slowOrchChecker.BeginSegment()
```

Comment:
I believe this only measures the length of the input segment. It doesn't actually measure how long it took to send; that's what the `took` field does below.

Note that the segment length doesn't really matter. I have tested with segments of 10+ seconds and that works fine, in fact the minimum is [easily adjustable](https://github.com/livepeer/go-livepeer/blob/e6aac56deb788c5528bf335332e6d0227041233b/media/whip_server.go#L190).

The problem is when we are sending segments at a slower rate than we are producing them. That's what the SlowOrchestrator check is for.

Reply:
Ok, I see. Let me revert this part. I'll just leave the `took` part.